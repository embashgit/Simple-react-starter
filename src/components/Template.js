import React, { Component } from 'react'

export class Template extends Component {
    render() {
        return (
            <div>
                <p>Hi Template</p>
            </div>
        )
    }
}

export default Template
