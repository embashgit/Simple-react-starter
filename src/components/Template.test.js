import React from 'react';
import { shallow } from 'enzyme';
import Template from './Template';
describe('Template', () => {
  it('should render correctly in "debug" mode', () => {
    const template = shallow(<Template debug />);
  
    expect(template).toMatchSnapshot();
  });
});