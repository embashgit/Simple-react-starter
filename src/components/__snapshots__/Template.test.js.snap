// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template should render correctly in "debug" mode 1`] = `
<div>
  <p>
    Hi Template
  </p>
</div>
`;
